Examples/Windpower Example/readme.md

```diff
@@ -0,0 +1,17 @@
+# PySAM Windpower Module Example
+
+This example illustrates how to work with PySAM's Windpower module for wind turbines and wind farms.
+
```

Comment:
Can you add what version of PySAM and SAM this was created with, and what it was last tested with?

Reply:
Fixed in f4149970cc3a4f3134c0a4f51c542baf1af9cce6